platform/engineering/collab-cycle/architecture-intent-meeting.md

```diff
@@ -2,11 +2,11 @@
 
 __*** DRAFT *** DRAFT *** DRAFT ***__
 
-The Architecture Intent meeting helps your team build a solution that meets Veteran-facing Service Platform (VSP) engineering standards and lowers the potential for launch-blocking issues later in the development cycle.
+The Architecture Intent meeting helps your team build a solution that meets Veteran-facing Service Platform (VSP) engineering and security standards and lowers the potential for launch-blocking issues later in the development cycle.
```

Comment:
I believe the name was been changed from VSP to VA.gov Platform a while back. @andreahewitt-odd would know better for sure.